scoring.js

```diff
@@ -0,0 +1,56 @@
+const calculateScore = (player) => {
+  switch (player.position) {
+    case 'QB':
+      return calcPassingScore(player) + calcRushingScore(player)
+
+    case 'RB':
+      return calcRushingScore(player) + calcReceivingScore(player) + calcReturningKickScore(player) + calcReturningPuntScore(player)
+
+    case 'WR':
+      return calcRushingScore(player) + calcReceivingScore(player) + calcReturningKickScore(player) + calcReturningPuntScore(player)
```

Comment:
One neat trick you can use here since RB and WR are the same calculation is to use `switch`'s run-to-end behavior to your advantage.

```JavaScript
switch (player.position) {
  case 'QB':
    return calcPassingScore(player) + calcRushingScore(player)
  case 'RB':
  case 'WR':
    return calcRushingScore(player) +
      calcReceivingScore(player) +
      calcReturningKickScore(player) +
      calcReturningPuntScore(player)
  case 'TE':
    return calcReceivingScore(player)
  default:
    return 0
}
```

Since there is no break or return in the RB case, if it matches, it will immediately run the WR code which is what we'd want for RB too.